Packages.props

```diff
@@ -6,24 +6,23 @@
 		<PackageReference Update="System.Text.Json" Version="4.7.2" />
 		<PackageReference Update="xunit.assert" Version="[2.4.0, 3)" />
 		<PackageReference Update="xunit.extensibility.core" Version="[2.4.0, 3)" />
-		<PackageReference Update="System.Linq.Async" Version="[5.0.0, 6)" />
+		<PackageReference Update="System.Linq.Async" Version="[6.0.1, 7)" />
```

Comment:
We should make this make as compatible as possible ... [5.0.0, 7)

Reply:
c7ab170f7f23be841643e35c8a04d4f915787c9b